### DISCUSSION TITLE
ARM64 windows client

### DISCUSSION THREAD

**Discussion by sstudent** [uniuuu/zotprime#30] [General]
Hi,
I have another question, is there a chance to build a client application for ARM64 winsows based on the new zotero 7 https://forums.zotero.org/discussion/114768/available-for-beta-testing-zotero-for-windows-on-arm
to work on Windows Surface pro SQ3 na Copilot+?
thanks in advance.

**Comment by uniuuu** [comment]
> Hi, I have another question, is there a chance to build a client application for ARM64 winsows based on the new zotero 7 https://forums.zotero.org/discussion/114768/available-for-beta-testing-zotero-for-windows-on-arm to work on Windows Surface pro SQ3 na Copilot+? thanks in advance.

Hi @sstudent 
Great news!
I'll update you when have it ready.

**Comment by uniuuu** [comment]
@sstudent 
It's been released. Please try latest code and pull latest images with ARM64 support.
Thank you!
